src/localization.js

```diff
@@ -12,14 +12,15 @@ export default class Localization extends I18n {
     }
 
     selectAvailableLocale (defaultLocale) {
-
         let locale = this.formatLocale(window.navigator.userLanguage || window.navigator.language || DEFAULT_LOCALE);
 
         if (defaultLocale) {
             locale = this.formatLocale(defaultLocale);
         } else {
             if (this.locale) {
                 locale = this.formatLocale(this.locale)
+            } else {
```

Comment:
This checks if another method is setting a language, if not then it checks if a user is trying to set the language, if not, then it uses browser language.